### PR TITLE
Don't return an error if there are no records yet

### PR DESCRIPTION
**Pull request by dominikbraun**

### CODE REVIEW

File: core/timetrace.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func New(config *config.Config, fs Filesystem) *Timetrace {
 // Since parallel work isn't supported, the previous work must be stopped first.
 func (t *Timetrace) Start(projectKey string, isBillable bool) error {
 	latestRecord, err := t.LoadLatestRecord()
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrAllDirectoriesEmpty) {
```
Comment (joshuaherrera) [inline core/timetrace.go:59]: I tested with this and it still fails, but works with 
`if err != nil && !errors.Is(err, ErrTrackingNotStarted) {`

**Comment by dominikbraun** [comment]
Update: Instead if ignoring `ErrAllDirectoriesEmpty` in general, only `Start` ignores the error as @KonstantinGasser mentioned in #102.

Also, `LoadLatestRecord` now returns a potential `ErrAllDirectories` directly instead of an `ErrTrackingNotStarted`, because this error is without the scope of a function loading a record.

**Review by joshuaherrera — CHANGES_REQUESTED** [review]

File: core/timetrace.go (hunk 1 above)
Comment (dominikbraun) [reply core/timetrace.go:59]: Pretty strange since `LoadLatestRecord` should only return an `ErrAllDirectoriesEmpty` (from latestNonEmptyDir`) 🤔

Comment (joshuaherrera) [reply core/timetrace.go:59]: Let me try again, I might have missed something.

Comment (dominikbraun) [reply core/timetrace.go:59]: When I check of `!errors.Is(err, ErrTrackingNotStarted)`, I get an error:

```
❗ Failed to start tracking: all directories empty
```

This indicates that `ErrAllDirectoriesEmpty` is returned.

Comment (KonstantinGasser) [reply core/timetrace.go:59]: checking for the `ErrAllDirectoriesEmpty` works fine for me too if `LoadLatestRecord` returns the error straight from `latestNonEmptyDir`

**Review by joshuaherrera — APPROVED** [review]
Tested the changes and it works.

**Comment by dominikbraun** [comment]
Thanks @joshuaherrera and @KonstantinGasser!

File: core/timetrace.go (hunk 1 above)
Comment (joshuaherrera) [reply core/timetrace.go:59]: @KonstantinGasser @dominikbraun  Right, this was my mistake, I hadn't updated my record.go file, apologies.
